data/filedefs/filetypes.HCL.conf

```diff
@@ -0,0 +1,38 @@
+# For complete documentation of this file, please see Geany's main documentation
+
+# This custom filetype uses the HTML styling settings and the Zephir lexer filetype
+# for pretty good results.
+# Symbols are parsed using a Terraform specific ctags parser.
+# For additional autocompletion tags, see https://wiki.geany.org/tags/start#hcl_terraformopentofu_tags.
+[styling=HTML]
+
+[keywords]
+# all items must be in one line
+php=data locals module output provider resource terraform variable
+
+[settings=PHP]
+extension=tf
+lexer_filetype=Zephir
+tag_parser=Terraform
+comment_single=#
+comment_use_indent=true
+mime_type=
+
+[indentation]
+width=2
+type=0
```

Comment:
Are those mandatory? Otherwise we usually don't enforce it in the filetype files.

Reply:
Not mandatory, just common. I commented them out as we do for other filetypes. Still, I kept the provided values at 2 spaces which is very common for Terraform files.